models/news-models.js

```diff
@@ -45,8 +46,43 @@ const fetchArticlesById = (article_id) => {
     })
 }
 
+const checkCommentExists = (article_id) => {
+
+    let queryString = `
+    SELECT article_id FROM articles
+    WHERE articles.article_id=$1
+    `;
+
+    return db.query(queryString, [article_id]).then((response) => {
+        if (response.rowCount === 0) {
+            return Promise.reject({status: 404, msg: 'article_id does not exist'});
+        } else {
+            return response.rows;
```

Comment:
If you are using this as a conditional check then it doesn't need to return data. 
Maybe you could refactor this to return a boolean? 

Then your rejected promise could be in the controller, and since you are translating the abcence of retruned rows to the client as a 404, that would make more sense from an archetectural point of view in tearms of MVC.